test/integration-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+make localnet-stop || exit 1
+rm -rf build || exit 1
```

Comment:
AFAIR, "rm -rf" never returns exit code != 0, so I think you can safely remove the "|| exit 1"